src/panels/lovelace/cards/hui-markdown-card.ts

```diff
@@ -27,6 +27,8 @@ export class HuiMarkdownCard extends LitElement implements LovelaceCard {
   }
 
   @property() private _config?: MarkdownCardConfig;
+  @property() private content?: string = "";
+  @property() private connection: (() => void) | null = null;
 
   public getCardSize(): number {
     return (
```

Comment:
Let's allow overriding this from the config too. It is kinda useless, especially with templates.